### DISCUSSION TITLE
filecoin rename

### DISCUSSION THREAD

**Discussion by likhita-8091** [filecoin-project/FIPs#261] [Enhancements - Organizational]
Whether the coin word in the project name can be removed is sensitive. Basically, no blockchain project adds coin to its own token.

**Comment by kaitlin-beegle** [comment]
Hey @CodingJzy! Interesting idea, and a good point. 

However, the 'Issues' forum is now reserved for active FIPs moving towards network acceptance in the current upgrade cycle.  I'm going to migrate this topic to the Governance Discussion Forum for others to weigh in on. Let me know if you have any questions!

**Reply by likhita-8091** [reply]
No problem
